src/daemon/ca/events.rs

```diff
@@ -970,6 +974,13 @@ impl fmt::Display for CaEvtDet {
 
             // Route Authorizations
             CaEvtDet::RouteAuthorizationAdded { auth } => write!(f, "added ROA: '{}'", auth),
+            CaEvtDet::RouteAuthorizationComment { auth, comment } => {
+                if let Some(comment) = comment {
+                    write!(f, "added comment to ROA: '{}' => {}", auth, comment)
+                } else {
+                    write!(f, "removed comment from ROA: '{}'", auth)
```

Comment:
Is this a removal if the ROA authorization had no comment to start with? Or is it assumed that we can't get to this point with a None comment if the ROA authorization didn't already have a comment?

Reply:
The update is only accepted if there is a change. So this could only be confusing if the comment was an empty string I suppose and then be removed. But it would still be accurate then, as written.